package.json

```diff
@@ -134,6 +135,7 @@
     "reflect-metadata": "^0.2.2",
     "sqlite3": "^5.1.7",
     "string-template": "^1.0.0",
+    "string-width": "^7.2.0",
```

Comment:
@EnzoDPrado Why did you install this dependency? I didn't see it in use in your changes.

Reply:
Because the jest-cli have a dependency in this library, and it is crashing when the test run because of it.

If you run yarn and access this folder:
node_modules/jest-cli/node_modules/cliui/build/index.csj
and look at the line 291, you will see this line:
'const stringWidth = require('string-width');'

Before my alterations, the require was appointing for no where, and this was crashing the test running.

Reply:
Got it, move this dependency to devDependencies. jest doesn't run in a productive environment

Reply:
Have you tried updating it? It's not common to install a dependency of your dependency.